.github/chronographer.yml

```diff
@@ -1,30 +1,10 @@
----
-
+branch-protection-check-name: Changelog entry
 action-hints:
-  # check-title-prefix: chng  # default: `{{ branch-protection-check-name }}: `
-  external-docs-url: https://pip.pypa.io/how-to-changelog
+  check-title-prefix: "Chronographer: "
+  external-docs-url: https://pip.pypa.io/dev/news-entry-failure
   inline-markdown: >
-    Check out https://pip.pypa.io/how-to-changelog
-
-branch-protection-check-name: Timeline protection
-
+    See https://pip.pypa.io/dev/news-entry-failure for details.
 enforce-name:
-  # suffix: .md
   suffix: .rst
-
-exclude:
-  bots:
-  - dependabot-preview
-  - dependabot
-  - patchback
```

Comment:
@pradyunsg FTR if the bots are not listed, _any_ GitHub App created PR causes a neutral check. With an explicit list, only the ones listed behave like that and other apps are considered "humans" for whom the change note requirement applies.

Your change is correct, I'm just documenting the effect for history.